File: Application/Services/Operations/Main/Partners/Dtos/PartnerPaymentPixDto.cs


using Application.Services.Shared.Dtos;

namespace Application.Services.Operations.Main.Partners.Dtos
{
    public class PartnerPaymentPixDto : RootBaseDto
    {
        public string Key { get; set; }
        public string Value { get; set; }
        public string Holder { get; set; }
        public int PaymentDataId { get; set; }
        public PaymentDataDto PaymentData { get; set; }
    }
}

File: Views.old/src/components/main/partner/commons-components/dtos/partner-payment-pix-dto.ts
import { PaymentDataDto } from "./payment-data-dto";

export class PartnerPaymentPixDto {
  id: number;
  key: string;
  value: string;
  holder: string;
  deleted: boolean;
  paymentDataId: number;
  paymentData: PaymentDataDto;

}
